pwa/src/components/dashboardCard/DashboardCard.tsx

```diff
@@ -10,33 +10,59 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { dateTime } from "../../services/dateTime";
 import { faArrowRight, faTrash } from "@fortawesome/free-solid-svg-icons";
 import { getStatusTag } from "../../services/getStatusTag";
+import { Button } from "../button/Button";
 
 export interface DashboardCardProps {
   title: {
     label: string;
     href: string;
   };
   type: string;
-  status?: string | boolean;
-  isEnabled?: boolean | undefined;
-  lastRun?: string;
-  lastCall?: string;
-  onDelete?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.TouchEvent<HTMLButtonElement>) => void;
-}
+  tags: {
+    label: string;
+    tooltip: string;
+  }[];
```

Comment:
Since we're reusing this object a couple of times:

1. Right here;
2. As the return statement of the `createTags` function;
3. As the type of `tagsArray` _within_ the `createTags` function.

I think we should make this into a actual type, like so:

```
type TDashboardCardTag = { label: string, tooltip: string };
```

By doing this, we only have to write the "structure" of this tag once, but we can implement it everywhere we need it.

```
export interface DashboardCardProps {
  ...
  tags: TDashboardCardTag[]; // <= NOTE: we're making it into an array here
  ...
}

... 

const generateTagsFromDashboardCard = (dashboardCard: any): TDashboardCardTag[] => {
  ...
}
```